openstack/objectstorage/v1/swauth/requests.go

```diff
@@ -0,0 +1,70 @@
+package swauth
+
+import "github.com/gophercloud/gophercloud"
+
+// AuthOptsBuilder describes struct types that can be accepted by the Auth call.
+// The AuthOpts struct in this package does.
+type AuthOptsBuilder interface {
+	ToAuthOptsMap() (map[string]string, error)
+}
+
+// AuthOpts specifies an authentication request.
+type AuthOpts struct {
```

Comment:
`:facepalm:`

I had that, then started renaming things to `Authenticate`, realized I liked `Auth` better, did `git checkout .` etc etc... hang on :)